mono/mini/llvm-jit.cpp

```diff
@@ -466,39 +464,6 @@ mono_llvm_dispose_ee (MonoEERef *eeref)
 {
 }
 
-MonoCPUFeatures
```

Comment:
Moved `mono_llvm_get_cpu_features ` from llvm-jit.cpp to mini-llvm.cpp (because it's not JIT-specific and also can be used in AOT (use_current_cpu=true)